### DISCUSSION TITLE
Unable to buy Teams plan

### DISCUSSION THREAD

**Discussion by aceclouddev** [community/community#132200] [New to GitHub]
Hello there, 

I am getting "special characters in the email address" error while buying the Teams plan of GitHub. 

Here is the screenshot of the issue: 

![image](https://github.com/user-attachments/assets/ca29d3f7-5074-433f-9ebc-6302af9ad4c6)

I have tried multiple emails, but nothing is working. 

Please help me resolve this issue. 

Thanks

**Comment by ghostinhershell** [comment]
Hi @aceclouddev! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
